themes/grav/app/main.js

```diff
@@ -61,10 +48,6 @@ export default {
     Dashboard,
     Pages,
     Forms,
-    Scrollbar: {
-        Scrollbar,
-        Instance: contentScrollbar
-    },
```

Comment:
I would leave this as empty objects for backward compatibility. Less prone to js failing.